regressiontests/stable-interact/src/test/java/org/apache/causeway/testdomain/interact/ActionInteractionTest_usingAllowSafeSemantics.java

```diff
@@ -0,0 +1,107 @@
+/*
+ *  Licensed to the Apache Software Foundation (ASF) under one
+ *  or more contributor license agreements.  See the NOTICE file
+ *  distributed with this work for additional information
+ *  regarding copyright ownership.  The ASF licenses this file
+ *  to you under the Apache License, Version 2.0 (the
+ *  "License"); you may not use this file except in compliance
+ *  with the License.  You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+package org.apache.causeway.testdomain.interact;
+
+import javax.inject.Named;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.stereotype.Service;
+import org.springframework.test.context.TestPropertySource;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.apache.causeway.applib.Identifier;
+import org.apache.causeway.applib.annotation.PriorityPrecedence;
+import org.apache.causeway.applib.annotation.SemanticsOf;
+import org.apache.causeway.applib.annotation.Where;
+import org.apache.causeway.applib.services.iactnlayer.InteractionContext;
+import org.apache.causeway.core.config.presets.CausewayPresets;
+import org.apache.causeway.core.security.authorization.Authorizor;
+import org.apache.causeway.testdomain.conf.Configuration_headless;
+import org.apache.causeway.testdomain.model.interaction.Configuration_usingInteractionDomain;
+import org.apache.causeway.testdomain.model.interaction.InteractionDemo;
+import org.apache.causeway.testdomain.util.interaction.InteractionTestAbstract;
+
+import lombok.val;
+
+@SpringBootTest(
+        classes = {
+                Configuration_headless.class,
+                Configuration_usingInteractionDomain.class,
+                ActionInteractionTest_usingAllowSafeSemantics.AuthorizorDenyUse.class
+        },
+        properties = {
+                "causeway.security.allowActionsWithSafeSemanticsToBeInvokedWithOnlyViewingPermission=TRUE",
```

Comment:
Do we need one further test to confirm that when not set, then the behaviour still requires CHANGING perm?

Reply:
good point, lets add that